spec/script/import_scripts/base_spec.rb

```diff
@@ -60,4 +60,34 @@ def import_bookmarks
     MockSpecImporter.new(import_data).perform
     expect(SiteSetting.purge_unactivated_users_grace_period_days).to eq(0)
   end
+
+  describe "#create_post" do
+    let(:importer) { described_class.new }
+    fab!(:user) { Fabricate(:user) }
+    let(:post_params) {
+      {
+        user_id: user.id,
+        raw: "Test post [b]content[/b]",
+        title: "Test topic for post"
+      }
+    }
+
+    it "creates a Post" do
+      expect {
+        importer.create_post(post_params, 123)
+      }.to change{Post.count}.by(1)
+    end
+
+    if ENV["IMPORT"] == "1"
```

Comment:
It's a little unconventional to do conditional testing like this (developers would need to prefix their rspec command with `IMPORT=1` to activate these tests) but then it's a little unconventional to do conditional gem includes in Gemfile.

Arguably it would be more flexible if the condition was `String.method_defined?(:bbcode_to_md)` (Then developers need only have the gem installed. No need to set `IMPORT=1`). Let me know if you'd prefer that, but I decided it would look even more unclear as to why these tests are only sometimes run. As it stands, the first test here serves to document the unusual conditional Gemfile set-up.